provisioning/requirements.yml

```diff
@@ -36,7 +36,7 @@
 - src: geerlingguy.nodejs
   version: 3.2.0
 - src: geerlingguy.php
-  version: 3.2.1
+  version: 3.2.2
```

Comment:
Good catch! I thought I had updated the role in requirements... but apparently not.